src/v2/view-builder/views/ov/SwitchEnrollChannelLinkView.js

```diff
@@ -7,7 +7,7 @@ export default View.extend({
   template: hbs `
     {{#if isQRcodeChannel}}
       <a href="#" class="switch-channel-link">
-        {{{i18n code="oie.enroll.okta_verify.qrcode.cannotScan" bundle="login"}}}</a>
+        {{{i18n code="enroll.totp.cannotScan" bundle="login"}}}</a>
```

Comment:
String within brackets is customized `Cant Scan `

Before: 

![Screen Shot 2021-05-26 at 11 03 18 AM](https://user-images.githubusercontent.com/23267876/119745924-0cfea280-be44-11eb-8c96-4a70a775652c.png)



After: 


![Screen Shot 2021-05-26 at 5 01 00 PM](https://user-images.githubusercontent.com/23267876/119745900-02440d80-be44-11eb-9025-ccd3165e6c84.png)